Makefile

```diff
@@ -32,6 +35,20 @@ ut: dist/calico_kubernetes
 	'/tmp/etcd -data-dir=/tmp/default.etcd/ >/dev/null 2>&1 & \
 	PYTHONPATH=/code/calico_kubernetes nosetests calico_kubernetes/tests -c nose.cfg'
 
+# UT runs on Cicle need to create the calicoctl binary
+ut-circle: kubernetesbuild.created dist/calico_kubernetes
+	# Can't use --rm on circle
+	# Circle also requires extra options for reporting.
+	docker run \
+	-v `pwd`/calico_kubernetes:/code/calico_kubernetes \
+	-v $(CIRCLE_TEST_REPORTS):/circle_output \
+	-e COVERALLS_REPO_TOKEN=$(COVERALLS_REPO_TOKEN) \
+	calico/kubernetes-build bash -c \
+	'/tmp/etcd -data-dir=/tmp/default.etcd/ >/dev/null 2>&1 & \
```

Comment:
We don't need this bit about running etcd - the kubernetes plugin tests don't rely on etcd being present / running.